joomla/joomla-cms
# [4.0] Make transitions translateable

Pull Request for Issue #35094 .

### Summary of Changes
Transitions are now translateable everywhere.

For example:

Backend article edit view:
![grafik](https://user-images.githubusercontent.com/1229869/131603054-2c925b21-3caf-4f90-8b27-c39dcd1cdf25.png)

Backend article list view:
![grafik](https://user-images.githubusercontent.com/1229869/131603118-4698d739-ac60-4941-94b8-024cbf462f66.png)

### Testing Instructions
- Make a fresh install of this PR
- Activate the workflow
- Go to the transitions
- See that there are now language keys as title
- Check the different occurrence of the transitions (backend/frontend edit article, list view backend)
- Use own language keys in transitions + language override function.


### Actual result BEFORE applying this Pull Request
Transitions are not (everywhere) translated


### Expected result AFTER applying this Pull Request
Transitions are translated everywhere.
